javascript-conditionals/main.js

```diff
@@ -0,0 +1,149 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+function isUnderFive(num) {
+  if (num < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+console.log('isUnderFive(4) value:', isUnderFive(4));
+console.log('isUnderFive(10) value:', isUnderFive(10));
+
+function isEven(num) {
+  if (num % 2) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
You either need to add `=== 0` or swap your `return` statements. You currently have this backwards. So `9` is technically `return`ing `true` rather than `false`

Reply:
Thank you Shawn! Updated isEven function is committed.